### DISCUSSION TITLE
Won't open in Ubuntu 20.04 LTS

### DISCUSSION THREAD

**Discussion by kradeelav** [Tyrrrz/DiscordChatExporter#1292] [Q&A]
Hello! 

I've extracted DCE (GUI interface option) and clicked on the discordchatexporter icon but it doesn't open and gives me the usual "Could Not Display Discord Chat Explorter / there are no applications...." error. What am I doing wrong?  

I've already made sure that "execute as program" checkbox is checked

**Comment by Tyrrrz** [comment]
Can you `ls` your working directory and copy-paste here?
